Makefile

```diff
@@ -7,6 +7,11 @@ help: ## Show this help screen
 	@echo ''
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+# Container build options
+IMAGE_REPO=quay.io/operator-framework/k8s-provisioner
```

Comment:
Note: we need to create this quay repository still.

Reply:
I have went ahead and created it.

> what happens when we have a second image?
That's a good point, I think our Makefile/repository structure will have to change